index.js

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
That's not a correct fix. You can't just add this to a non-binary.

The correct fix is to add a inline `// eslint-disable-line unicorn/no-process-exit` comment.